packages/@ember/map/lib/ordered-set.js

```diff
@@ -166,3 +164,14 @@ export default class OrderedSet {
     return set;
   }
 }
+
+export default class OrderedSet extends __OrderedSet__ {
+  constructor() {
+    super();
+    deprecate('Use of @ember/OrderedSet is deprecated. Please use native `Map` instead', false, {
```

Comment:
maybe it should say, Please use native `Map` or  ember-ordered-set addon instead ?

Reply:
or "Please use native `Map` or `https://github.com/emberjs/ember-ordered-set` addon instead"

Reply:
This is a good idea and can be a followup PR